### PR TITLE
Fix bug with Travis jobs not failing with correct exitcode #404

### PR DESCRIPTION
**Pull request by denverwilliams**
# Fix bug with Travis jobs not failing with correct exitcode #404 

## Description
Fix bug with Travis jobs not failing with correct exitcode.

## Issues:
Refs: cncf/cnf-conformance/issues/404

## How has this been tested:
 - [ ] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [ ] Tasks in issue are checked off

### CODE REVIEW

File: src/tasks/platform/observability.cr [hunk 1]
```diff
@@ -66,6 +66,14 @@ namespace "platform" do
...
       install_cri_tools = `kubectl create -f cri_tools.yml`
+      pod_ready = ""
+      pod_ready_timeout = 45
+      until (pod_ready == "true" || pod_ready_timeout == 0)
+        pod_ready = CNFManager.pod_status("cri-tools").split(",")[2]
+        puts "Pod Ready Status: #{pod_ready}"
+        sleep 1
+        pod_ready_timeout = pod_ready_timeout - 1
+      end
       cri_tools_pod = CNFManager.pod_status("cri-tools").split(",")[0]
```
Comment (taylor) [inline src/tasks/platform/observability.cr:77]: @denverwilliams Should these be in a different PR for #403 or a different ticket?

File: tools/ephemeral_env/ephemeral_env.cr [hunk 2]
```diff
@@ -123,7 +123,7 @@ elsif ARGV[0]? && ARGV[0] == "create_env"
     puts "Required argument missing [-n, --name]"
   else
     puts "Creating ENV For: \n Name: #{env_name} \n Kubeconfig: #{kubeconfig}"
-    `docker run --name #{env_name} --network host -d -e GITHUB_USER=$GITHUB_USER -e GITHUB_TOKEN=$GITHUB_TOKEN -v $(pwd):/cnf-conformance -v #{kubeconfig}:/root/.kube/config -ti cnf-test /bin/sleep infinity`
+    `docker run --name #{env_name} --network host -d -e GITHUB_USER=$GITHUB_USER -e GITHUB_TOKEN=$GITHUB_TOKEN -e DOCKERHUB_USERNAME=$DOCKERHUB_USERNAME -e DOCKERHUB_PASSWORD=$DOCKERHUB_PASSWORD -v $(pwd):/cnf-conformance -v #{kubeconfig}:/root/.kube/config -ti cnf-test /bin/sleep infinity`
```
Comment (taylor) [inline tools/ephemeral_env/ephemeral_env.cr:126]: Is this about the travis exit code issue? I recall the ephemeral env being mentioned.

File: src/tasks/platform/observability.cr [hunk 3]
```diff
@@ -75,7 +83,7 @@ namespace "platform" do
       LOGGING.info "container_repo_digests: #{repo_digest_list}"
       id_sha256_list = repo_digest_list.reduce([] of String) do |acc, repo_digest|
         LOGGING.debug "repo_digest: #{repo_digest}"
-        cricti = `kubectl exec -ti #{cri_tools_pod} crictl inspecti #{repo_digest}`
+        cricti = `kubectl exec -ti #{cri_tools_pod} -- crictl inspecti #{repo_digest}`
```
Comment (wavell) [inline src/tasks/platform/observability.cr:86]: We should have a new test that checks for crictl (CRI) compliance of the runtime since we are now using this crictl tool.  @taylor

**Review by wavell — APPROVED** [review]
